.github/workflows/check-folder-name.yml

```diff
@@ -0,0 +1,52 @@
+name: Rename Folder
+
+on:
+  pull_request:
+    paths:
+      - '**/rule.md'
+    types: [opened, synchronize]
+
+jobs:
+  rename-folder:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      repository-projects: read
+      pull-requests: write
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        token: ${{ secrets.GITHUB_TOKEN }}
+        ref: ${{ github.event.pull_request.head.ref }}
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+
+    - name: Check for modified rule.md
+      id: check_modified
+      run: |
+        if git diff --name-only $(git merge-base origin/main HEAD) | grep -q 'rule.md'; then
+          echo "modified=true" >> "$GITHUB_OUTPUT"
+        else
+          echo "modified=false" >> "$GITHUB_OUTPUT"
+        fi
+
+    - name: Rename folders if necessary
+      id: rename_folders
+      if: steps.check_modified.outputs.modified == 'true'
+      run: |
+        # chmod +x .workflow/rename-folders.sh
```

Comment:
Is this line run?

Reply:
ah I forgot to delete it, thanks for spotting this!